pom.xml

```diff
@@ -30,6 +30,7 @@
     <packaging>pom</packaging>
 
     <modules>
+        <module>test-reports</module>
```

Comment:
The module order is being changed to address an issue with the nexus staging plugin.
If the last module has a configuration to skip it's deployment, then the deployment of the whole project is skipped:

https://issues.sonatype.org/browse/NEXUS-9138